Extensions/DebuggerTools/JsExtension.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+    createExtension: function(_, gd) {
+      const extension = new gd.PlatformExtension();
+      extension.setExtensionInformation(
+        'DebuggerTools',
+        _('Debugger Tools'),
+        _(
+          'Allow to interact with the editor debugger from the game.'
+        ),
+        'Arthur Pacaud (arthuro555)',
+        'MIT'
+      );
+
+      extension
+      .addAction(
+        'Pause',
+        _('Pause game execution'),
+        _(
+          'This pauses the game, useful for inspecting the game state through the debugger. ' + 
+          'Note that events will be still executed until the end before the game is paused.'
+        ),
+        _('Pause game execution'),
+        _('Debugger Tools'),
+        'res/ribbon_default/bug32.png',
```

Comment:
Can you copy this image in `res/actions` for staying consistent in the usage of icons? Thanks!